lib/sen_tweet/metrics.ex

```diff
@@ -1,81 +1,50 @@
 defmodule SenTweet.Metrics do
```

Comment:
This module is the mentioned boilerplate code. **All** metrics run through this module but we're only doing stuff with the bitfeels metrics.